README.md

```diff
@@ -41,114 +41,113 @@ cyrusauth:
 
 If this is disabled, you get the buffer playback only once and then it is
-deleted. If this is enabled, the buffer is not deleted. This may be useful if
-you regularly use more than one client to connect to ZNC.
+deleted. If this is enabled, the buffer is not deleted. This may be useful
+if you regularly use more than one client to connect to ZNC.
 
 ## Using ZNC
 
-Once you have started ZNC you can connect with your favorite IRC-client to ZNC.
-You should use `username:password` as the server password (e.g. `/pass user:pass`).
+Once you have started ZNC you can connect with your favorite IRC-client to
+ZNC. You should use `username:password` as the server password (e.g.
+`/pass user:pass`).
 
 Once you are connected you can do `/msg *status help` for some commands.
-Every module you have loaded (`/msg *status listmods`) should additionally provide
-	/msg *modulename help
+Every module you have loaded (`/msg *status listmods`) should additionally
+provide `/msg *modulename help`
 
 ## File Locations
 
 In its data dir (`~/.znc` is default) ZNC saves most of its data. The only
-exception are modules and module data, which are saved in `<prefix>/lib/znc`
-and `<prefix>/share/znc`, and the znc binary itself.
+exception are modules and module data, which are saved in
+`<prefix>/lib/znc` and `<prefix>/share/znc`, and the znc binary itself.
 More modules (e.g. if you install some later) can be saved in
 `<data dir>/modules` (-> `~/.znc/modules`).
 
 In the datadir are only two files:
 
 - `znc.pid` - The pid of the currently running ZNC instance.
-- `znc.pem` - This is the server certificate ZNC uses for listening and is created
- with `znc --makepem`.
+- `znc.pem` - This is the server certificate ZNC uses for listening and is
+created with `znc --makepem`.
 
 These directories are also in there:
 
-- configs - Contains `znc.conf` (ZNC's config file) and backups of older configs.
+- configs - Contains `znc.conf` (ZNC's config file) and backups of older
+  configs.
 - modules - ZNC also looks in here for a module.
 - moddata - Global modules save their settings here.
   (e.g. webadmin saves the current skin name in here)
-- users   - This is per-user data and mainly contains just a moddata directory.
+- users   - This is per-user data and mainly contains just a moddata
+  directory.
 
 ## ZNC's config file
 
 This file shouldn't be too hard too understand. An explanation of all the
-items can be found on the [Configuration](http://wiki.znc.in/Configuration)-Page.
-Warning: better not to edit config, while ZNC is running.
+items can be found on the
+[Configuration](http://wiki.znc.in/Configuration)-Page.
+**Warning: better not to edit config, while ZNC is running.**
 
-To rehash the config file, you can send ZNC SIGHUP via
-	pkill -SIGHUP znc
-or you can login to ZNC and use
-	/msg *status rehash
+To rehash the config file, you can send ZNC SIGHUP via `pkill -SIGHUP znc`
+or you can login to ZNC and use `/msg *status rehash`
```

Comment:
Removed. The saving part stays untouched as you said nothing about it and it's not the same paragraph.